projects/plugins/boost/tests/e2e/package.json

```diff
@@ -14,7 +14,7 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
-		"build": "pnpm jetpack build plugins/jetpack plugins/boost packages/plugin-deactivation -v --no-pnpm-install --production",
+		"build": "pnpm jetpack build plugins/jetpack plugins/boost js-packages/image-guide packages/plugin-deactivation -v --no-pnpm-install --production",
```

Comment:
Do we need to be building the package? Wouldn't it get built when we build Boost?

Reply:
I couldn't get this to work properly without building it on install. When I set `main` in `package.json` to a `ts` file and tried to import this package, it couldn't find it. I found I had to build a js file and point `main` at that. 🤔 Maybe we can revisit this?